jl4-lsp/src/LSP/L4/Viz/Ladder.hs

```diff
@@ -1,24 +1,43 @@
 {-# LANGUAGE ViewPatterns #-}
-
-module LSP.L4.Viz.Ladder where
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+{-# HLINT ignore "Redundant lambda" #-}
+
```

Comment:
```suggestion
```

there are hlint hints all over the place, shouldn't be starting to ignore them on a per file basis now